test/parallel/test-child-process-detached.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
-require('../common');
+var common = require('../common');
```

Comment:
It's a nit so you can ignore it, of course, but if you do want to const-ify `common` here, you can also do it in the other files too. (Looks like you used `var` where the file was doing that for other modules and `const` where it was already using that. Which is also just fine. Like I said: nit.)